soc/arm/nuvoton_npcx/common/scfg.c

```diff
@@ -22,29 +22,13 @@ struct npcx_scfg_config {
 	uintptr_t base_glue;
 };
 
-/*
- * Get io list which default functionality are not IOs. Then switch them to
- * GPIO in pin-mux init function.
- *
- * def_io_conf: def-io-conf-list {
- *               compatible = "nuvoton,npcx-pinctrl-def";
- *               pinctrl-0 = <&alt0_gpio_no_spip
- *                            &alt0_gpio_no_fpip
- *                            ...>;
- *               };
- */
-static const struct npcx_alt def_alts[] =
-			NPCX_DT_IO_ALT_ITEMS_LIST(nuvoton_npcx_pinctrl_def, 0);
-
-static const struct npcx_lvol def_lvols[] = NPCX_DT_IO_LVOL_ITEMS_DEF_LIST;
```

Comment:
Can support for the `def-lvol-io-list` node be left in until we get the Chromebook platforms updated.
This would allow the PR to get merged without breaking existing Chromebooks.  We can then migrate Chromebooks to use the `low-voltage-input` property, and then delete all code related to `def-lvol-io-list`.

Reply:
Sure, I will keep all of them. Once they are deprecated, add the other PR to remove them.

Reply:
Done.